sig-etcd/README.md

```diff
@@ -90,6 +90,10 @@ Distributed reliable key-value store for the most critical data of a distributed
 etcd manager
 - **Owners:**
   - [kubernetes-sigs/etcd-manager](https://github.com/kubernetes-sigs/etcd-manager/blob/main/OWNERS)
+### etcd-operator
+future site of the new etcd operator, currently used for wg-etcd-operator artifacts
```

Comment:
```suggestion
The official Kubernetes operator for etcd.
```

Reply:
Presumably that only becomes true when we finish the build and it turns into the etcd-operator subproject.